### ISSUE TITLE
Wrong macOS version shown in "About muCommander"

### ISSUE THREAD

**Issue by Andreas0602** [mucommander/mucommander#687]
The macOS version shown in "About muCommander" is wrong.

<img width="653" alt="Bildschirmfoto 2022-07-06 um 22 02 57" src="https://user-images.githubusercontent.com/79788982/177634518-d2dc1871-48ed-4338-b732-f693a8cb8c80.png">

**Comment by ahadas** [comment]
right, that's how Java detects it (System.getProperty("os.version"))
it might be fixed in a more advanced version of Java but it doesn't worth the effort to do that for this issue so once we update Java, we can check to see if this is solved

**Comment by andrewcrook** [comment]
I think it is most likely the version of JAVA but seems bad that JAVA cannot just get this information directly from the OS.

However, if needs be a wrapper could be built around the following macOS command-lines,

```
> sw_vers
ProductName:	macOS
ProductVersion:	12.4
BuildVersion:	21F79
```

```
> arch 
arm64
```

**Comment by ahadas** [comment]
@Andreas0602 did you check if this issue still happens with [the updated dmg of v1.2.0](https://github.com/mucommander/mucommander/releases/download/1.2.0-1/muCommander-1.2.0-2.dmg) (that is bundled with Java 20)?
